cognite/client/data_classes/data_modeling/query.py

```diff
@@ -195,15 +208,37 @@ def __eq__(self, other: Any) -> bool:
 
 
 class NodeResultSetExpression(ResultSetExpression):
+    """Describes how to query for nodes in the data model.
+
+    Args:
+        from_ (str | None): Chain your result-expression based on this view.
+        filter (Filter | None): Filter the result set based on this filter.
+        sort (list[InstanceSort] | None): Sort the result set based on this list of sort criteria.
+        limit (int | None): Limit the result set to this number of instances.
+        through (list[str] | tuple[str, str, str] | None): Chain your result-expression through this view.
+            The tuple must be on the form (space, view/version, property).
+        direction (Literal["outwards", "inwards"]): The direction to use when traversing direct relations.
+            Only applicable when through is specified.
+        chain_to (Literal["destination", "source"]): Control which side of the edge to chain to.
+            See below for more information.
+
+    The chain_to option is only applicable if the view referenced in the from field consists of edges.
```

Comment:
This should be "result expression" not "view". Looks like a bug in the service contract.
```suggestion
    The chain_to option is only applicable if the result rexpression referenced in `from` contains edges.
```